assignments/callbacks.js

```diff
@@ -27,29 +27,61 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+
+  return cb(arr.length)
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr.length-1)
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  return cb(x + y)
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  return cb(x*y)
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  if(list.includes(`${item}`) === true) {
```

Comment:
If statements are, be default, true/false statements. So you could also write it like this:
```suggestion
  if(list.includes(`${item}`)) {
```
But the way you wrote it works too!